cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -162,7 +178,12 @@ void Run(const std::vector<string>& bag_filenames) {
         break;
       }
 
-      // TODO(damonkohler): Republish non-conflicting tf messages.
+      // TODO(damonkohler): Check if republished tf messages are in conflict
```

Comment:
Ok, I will add a flag. What exactly would be a conflicting transform? Is it the following?

- a transformation which has Cartographer's `published_frame` as a child
- in case `provide_odom_frame` is true, a transformation which has `odom_frame` as child

Would it be enough to run a loop through the transforms in the message and do not publish them if they match one of the above, or did you have something else in mind?